### PR TITLE
issue-651

### PR DESCRIPTION
**Pull request by RobertSzkutak**
#651

### CODE REVIEW

File: deploy/lib/server_config.rb [hunk 1]
```diff
@@ -1784,9 +1784,9 @@ def capture_environment_config(full_config)
     raise ExitException.new("Capture requires the target environment's hostname to be defined") unless @hostname.present?
 
     if (full_config == nil)
-      databases = quote_arglist(find_arg(['--databases']) || "#{@properties["ml.content-db"]},#{@properties["ml.modules-db"]},#{@properties["ml.triggers-db"]},#{@properties["ml.schemas-db"]},#{@properties["ml.app-modules-db"]}")
+      databases = quote_arglist(find_arg(['--databases']) || "#{@properties["ml.content-db"]},#{@properties["ml.modules-db"]},#{@properties["ml.triggers-db"]},#{@properties["ml.schemas-db"]},#{@properties["ml.modules-db"]}")
```
Comment (dmcassel) [inline deploy/lib/server_config.rb]: `#{@properties["ml.modules-db"]}` is now repeated

File: deploy/lib/server_config.rb [hunk 2]
```diff
@@ -1784,9 +1784,9 @@ def capture_environment_config(full_config)
...
       # TODO: take content-forests-per-host into account properly, just taking first by default
-      forests = quote_arglist(find_arg(['--forests']) || "#{@properties["ml.content-db"]},#{@properties["ml.content-db"]}-001-1,#{@properties["ml.modules-db"]},#{@properties["ml.triggers-db"]},#{@properties["ml.schemas-db"]},,#{@properties["ml.app-modules-db"]}")
+      forests = quote_arglist(find_arg(['--forests']) || "#{@properties["ml.content-db"]},#{@properties["ml.content-db"]}-001-1,#{@properties["ml.modules-db"]},#{@properties["ml.triggers-db"]},#{@properties["ml.schemas-db"]},,#{@properties["ml.modules-db"]}")
```
Comment (dmcassel) [inline deploy/lib/server_config.rb]: `#{@properties["ml.modules-db"]}` is now repeated

**Comment by dmcassel** [comment]
@RobertSzkutak just need those two things fixed and this is ready to merge

**Comment by RobertSzkutak** [comment]
@dmcassel I believe it should be fixed now. Thanks for catching that

**Pull request merged**